pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+import logging
+import os
+import sys
+from astropy.io import fits
+from ogadf_schema.irfs import AEFF_2D, EDISP_2D, PSF_TABLE, BKG_2D, RAD_MAX
+from pathlib import Path
+
+
+def test_eventdisplay_example(caplog):
+    """Runs the EventDisplay example script and check its output."""
+
+    caplog.set_level(logging.WARNING, logger="fits_schema")
+    ROOT_DIR = Path(__file__).parent.parent.parent
+
+    # run script and check that it doesn't crash
+    script_path = os.path.join(ROOT_DIR, "examples/calculate_eventdisplay_irfs.py")
+    subprocess.check_output([sys.executable, script_path])
+
+    # check that the output file exists and it's not empty
+    outname = os.path.join(ROOT_DIR, "pyirf_eventdisplay.fits.gz")
+    assert os.path.isfile(outname) and os.path.getsize(outname)
+
+    # open FITS file
+    outfile = fits.open(outname)
+
+    # known errors
+    dimensionality_error = "Dimensionality of rows is 0, should be 1"
+
+    # check that each HDU respects the OGADF schema
```

Comment:
Most importantly, we could no check that the sensitivity compared to eventdisplay is now:

```
ratio = sens_pyirf / sens_ed
assert np.all((ratio > 0.5) & (ratio < 2))
``` 
```

Reply:
ah yes now we can add this here